composer.json

```diff
@@ -25,6 +25,7 @@
         "yiisoft/cookies": "^3.0@dev"
     },
     "require-dev": {
+        "infection/infection": "^0.16.3",
```

Comment:
Should the `infection` package be removed after all?

Reply:
Yeah. Now it should, I think. I've found no way to use it with this package :(